umap/static/umap/content.css

```
@@ -169,7 +169,7 @@ h2.tabs a:hover {
     background-color: #c0392b;
     color: #efefef;
     margin-top: 0;
-    padding: 14px 0;
+    padding: 1em;
```

We try to use CSS vars now to uniformize gaps and sizes:

```suggestion
    padding: var(--gutter);
```